odd-platform-ui/src/components/DataQuality/DataQualityFilters/DataQualityFilters.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Grid, Typography } from '@mui/material';
+import { Button } from 'components/shared/elements';
+import * as S from './DataQualityFiltersStyles';
+import { NamespaceFilter } from './FilterItem/NamespaceFilter';
+import { DatasourceFilter } from './FilterItem/DatasourceFilter';
+import { OwnerFilter } from './FilterItem/OwnerFIlter';
+import { TitleFilter } from './FilterItem/TitleFilter';
+import { TagFilter } from './FilterItem/TagFilter';
+import { useDataQualityContext } from '../DataQualityContext/DataQualityContext';
+
+export const DataQualityFilters: FC = () => {
+  const { t } = useTranslation();
+  const { updateFilter } = useDataQualityContext();
+
+  function clearTablesFilters() {
+    updateFilter('deNamespaceIds', []);
+    updateFilter('deDatasourceIds', []);
+    updateFilter('deOwnerIds', []);
+    updateFilter('deTitleIds', []);
+    updateFilter('deTagIds', []);
+  }
+
+  function clearTestsFilters() {
+    updateFilter('namespaceIds', []);
+    updateFilter('datasourceIds', []);
+    updateFilter('ownerIds', []);
+    updateFilter('titleIds', []);
+    updateFilter('tagIds', []);
+  }
```

Comment:
You can replace this by useCallback and useEffect for establish init filters state.
this is pseudo code. Can be some issues with types inffering for
```suggestion
  const [searchParams, setSearchParams] = useSearchParams();
  const initFilters = {
    deNamespaceIds: [],
    deDatasourceIds: [],
    ...
  }
  const resetFilters = useCallback(() => {
    setSearchParams({...searchParams, ...initFilters })
   }, [searchParams])
  
  useEffect(() => resetFilters(), [])
```